Dockerfile

```diff
@@ -15,25 +15,25 @@ RUN set -x \
     && apt-get update \
     && apt-get install curl -y \
     && curl -sL https://deb.nodesource.com/setup_6.x | bash - \
-    && apt-get install git postgresql-client bzip2 gcc g++ make libpq-dev libffi-dev nodejs --no-install-recommends -y \
+    && curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - \
+    && echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list \
```

Comment:
BTW, seems, `stable` refers to `yarn`, not to `debian`, they don't provide version-specific builds and the only alternative [dist](https://github.com/yarnpkg/releases/tree/gh-pages/debian/dists) is `rc`.

Reply:
Would installing it from sources be more predictable (and not so painful)?

Reply:
I mean, so we can pin onto a specific tag know to work in our environment.

Reply:
I think installing yarn "from sources", which actually means installing it with `npm` is quite suboptimal for various reasons:
* Insecure fetching of resources
* Broken handling of dependencies

Which are actually the reasons why we use `yarn` instead of `npm` in the first place.

Given the fact that the whole thing is shipped as a docker container, here what we are trying to do is just reduce the likelyhood of breakages not actually have a full bullet proof solution.

Also if you are doing this building often, since these are the beginning of the dockerfile without changing instructions before that you will most likely end up with this diff of the image cached.

Reply:
Alright. In any case, it would not probably break as long as we stay fairly current.